### DISCUSSION TITLE
Transcript Files Not Being Generated under macOS (Printing to Console Instead)

### DISCUSSION THREAD

**Discussion by Zyniker13** [openai/whisper#1782] [Q&A] [answered]
The title pretty much says it all: Under macOS (Sonoma 14.1.1), Whisper, via command line, is not generating transcript files even when they are specified, but, rather, is printing to the console.

Here is one of the commands I tried:

`whisper audio.m4a --model large -f all -o .`

**Comment by glangford** [comment]
(1) Make sure the current directory is writable. 
(2) The default output directory is the current directory; my guess is that '.' for the directory is not being expanded and you don't need to specify it anyway. Try

`whisper audio.m4a --model large -f all`

(3) In addition, the default is also to produce all outputs, so even more straightforward is

`whisper audio.m4a --model large`

and if you don't want console output 

`whisper audio.m4a --model large --verbose False`

**Reply by glangford** [reply, marked as answer]
p.s. Sonoma 14.1.1 works fine for me

**Reply by Zyniker13** [reply]
Going to mark your reply as an answer (in part because I do not know what fixed the issue), but:

After running a `brew` update, everything seems to be working again.
